docs/advanced/data_types.mdx

```diff
@@ -72,6 +73,14 @@ To supply your own QnA pair, use the data_type as `qna_pair` and enter a tuple.
 app.add_local('qna_pair', ("Question", "Answer"))
 ```
 
+### Doc file
+
+To add any doc/docx file, use the data_type as `docx`. Eg:
+
+```python
+app.add_local('docx', 'a_local_docx_file_path')
```

Comment:
this is not correct.
`add_local` is meant to add text which is in a Python variable. like string.

We should remove this.

Reply:
I think `add` remote is a URL string, and `add_local` is anything that's not a URL. QnA Pair also isn't a string.

Turns out, docx can do both:

> Docx2txtLoader: Defaults to check for local file, but if the file is a web path, it will download it to a temporary file, and use that, then clean up the temporary file after completion

So it can practically belong in both categories, we could argue that local file is the default so it should be under local, but I will move it to the remote section

Reply:
I expanded the documentation a bit, but moved it back to where you wanted it.